storage/mysql/storage_test.go

```diff
@@ -1271,9 +1271,9 @@ func ensureAllLeafHashesDistinct(leaves []trillian.LogLeaf, t *testing.T) {
 	// or pretty much any kind of usable data structures we could do this properly.
```

Comment:
I've renamed it to ensureAllLeavesDistinct() and clarified that we're using the leaf value hash for this.